chal6.html

```diff
@@ -0,0 +1,51 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8">
+    <title></title>
+  </head>
+  <body>
+    <input id = "input"><button onclick ="start()">Check</button>
+    <script type="text/javascript">
+      function start() {
+        var input =  document.getElementById("input").value;
+        var values = input.split("")
+        var ifIntOrFloat = Number.isInteger(values.length/2);
```

Comment:
Since this value isn't changing it can be a `const` as well. You can fix the `const` everywhere else.

Reply:
A better way to do this instead of using so many if statements is to actually change the starting position. Are you starting to double the numbers in index position `0` or `1`.